rdr_service/genomic/genomic_data_quality_components.py

```diff
@@ -105,32 +102,37 @@ def get_report_display_name(self):
 
         return display_name
 
-    def get_report_def(self, level, target, time_frame):
+    @staticmethod
+    def get_empty_report_string(display_name):
+
+        empty_report_string = f"No data to display for {display_name}"
+
+        return empty_report_string
```

Comment:
nit: can we just return the string here instead of instantiating the variablee

Reply:
fixed.